documentation/user-interface/main-menu.md

```diff
@@ -15,24 +15,24 @@ The main menu contains the following menus:
 * [Window](#window)
 * [Help](#help)
 
-> The set of available menu commands depends on an operating system.
+> The set of available menu commands depends on the operating system.
 
 ## TestCafe Studio
 
 The **TestCafe Studio** menu is available on MacOS and provides the following commands:
 
-* **About** - Provides TestCafe Studio's version and copyright.
-* **Services** - Allows you to work with other applications from TestCafe Studio.
+* **About** - Provides TestCafe Studio's version and copyright information.
+* **Services** - Allows you to work with other TestCafe Studio applications.
```

Comment:
It's incorrect. Not TestCafe Studio applications, other applications installed on your computer